NOTICE.md

```diff
@@ -0,0 +1,5 @@
+The source code developed for the LitmusChaos Project is licensed under Apache 2.0. 
+
+However, the LitmusChaos project contains unmodified/modified subcomponents from other Open Source Projects with separate copyright notices and license terms. 
```

Comment:
Let us call out the modified components explicitly. And is there a way that we can avoid those components?